doc/Indium.texi

```diff
@@ -342,7 +342,9 @@ chrome-devtools://inspector.html?...&ws=127.0.0.1:PORT/PATH
 @end example
 
 Evaluate @code{M-x indium-connect-to-nodejs RET 127.0.0.1 RET PORT RET PATH},
-@code{PORT} and @code{PATH} are the ones from the @cite{ws} parameter of the above URL.
+@code{PORT} and @code{PATH} are the ones from the @cite{ws} parameter of the above URL
```

Comment:
This file is generated, you should change the file in sphinx-doc/ instead, and optionally run `make texinfo` from the `sphinx-doc` folder.

Reply:
Ok, got it. Thanks.